scenarios/seir-stockflow/pyciemss/optimize-calibrate.json

```diff
@@ -0,0 +1,27 @@
+{
+  "engine": "ciemss",
+  "username": "not_provided",
+  "model_config_id": "seir-stockflow",
+  "dataset": {
+    "id": "optimize",
+    "filename": "optimize.csv"
+  },
+  "timespan": {
+    "start": 0,
+    "end": 19
+  },
+  "interventions": [
+    {
+      "timestep": 20,
+      "name": "beta"
```

Comment:
Is this name supposed to be different from the one in `optimize-simulation.json`?

Reply:
Just fixed this as well.